### PR TITLE
Fix issues in repo lens and message parsing

### PR DESCRIPTION
**Pull request by willscott**
* fills in missing modules in the repo lens chainstore / state manager apis
* catches panics possible in parsing messages
* makes message parsing less expensive api call

### CODE REVIEW

**Review by frrist — APPROVED** [review]

File: model/messages/parsedmessage.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ type ParsedMessage struct {
 	Value  string `pg:",notnull"`
 	Method string `pg:",notnull"`
 
-	Params []byte `pg:",type:jsonb,notnull"`
+	Params string `pg:",type:jsonb,notnull"`
```
Comment (frrist) [inline model/messages/parsedmessage.go:25]: This is the fix for the below errors, yeah?

```
2020-10-19T21:24:36.842Z        ERROR   message message/message.go:107  failed to process tipset        {"error": "process tipset: persist: persisting parsed messages: ERROR #22P02 invalid input syntax for type json", "height": 161014}
```

Comment (willscott) [reply model/messages/parsedmessage.go:25]: yes

File: tasks/message/message.go [hunk 2]
```diff
@@ -321,7 +330,17 @@ func parseMsg(m *messagemodel.Message, ts *types.TipSet, destCode string) (*mess
...
 		actor = statediff.LotusTypeUnknown
 	}
-	params, name, err := statediff.ParseParams(m.Params, int(m.Method), actor)
+	var params ipld.Node
+	var name string
+	var err error
+	go func() {
+		defer func() {
+			if r := recover(); r != nil {
+				err = fmt.Errorf("%v", r)
```
Comment (frrist) [inline tasks/message/message.go:339]: nit: use xerrors instead, we already import it here.

File: tasks/message/message.go [hunk 3]
```diff
@@ -321,7 +330,17 @@ func parseMsg(m *messagemodel.Message, ts *types.TipSet, destCode string) (*mess
...
+			}
+		}()
+		params, name, err = statediff.ParseParams(m.Params, int(m.Method), actor)
```
Comment (frrist) [inline tasks/message/message.go:342]: Is git diff being weird or is this (incorrectly) called in the anonymous go routine?

Comment (willscott) [reply tasks/message/message.go:341]: great catch! fixed.

File: tasks/message/message.go [hunk 4]
```diff
@@ -276,7 +278,13 @@ func (p *MessageProcessor) extractMessageModels(ctx context.Context, ts *types.T
...
 				return nil, nil, err
 			}
-			dstActor, err := p.node.StateGetActor(ctx, dstAddr, ts.Key())
+
+			st, err := state.LoadStateTree(p.node.Store(), ts.ParentState())
+			if err != nil {
+				return nil, nil, err
+			}
+
+			dstActor, err := st.GetActor(dstAddr)
```
Comment (frrist) [inline tasks/message/message.go:287]: Check this error

File: tasks/message/message.go [hunk 5]
```diff
@@ -335,6 +335,10 @@ func parseMsg(m *messagemodel.Message, ts *types.TipSet, destCode string) (*mess
 	var params ipld.Node
 	var name string
 	var err error
+
+	// TODO: the following closure is in place to handle the potential for panic
```
Comment (frrist) [inline tasks/message/message.go:339]: :heart:

**Review by frrist — APPROVED** [review]

**Pull request merged**